static/styles/portico/portico.css

```diff
@@ -151,7 +149,7 @@ html {
         height: calc(100vh - 59px);
 
         border-right: 1px solid hsl(219, 10%, 97%);
-        overflow: auto;
+        position: fixed;
```

Comment:
Make the sidebar fixes, so that the html scrollbar does not scrolls this and sidebar remains fixed. it has its own simplebar.